[2024-11-18T12:38:32.654+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T12:38:32.663+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_dag.dbt_test manual__2024-11-18T12:33:00.228556+00:00 [queued]>
[2024-11-18T12:38:32.668+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_dag.dbt_test manual__2024-11-18T12:33:00.228556+00:00 [queued]>
[2024-11-18T12:38:32.668+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-11-18T12:38:32.687+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): dbt_test> on 2024-11-18 12:33:00.228556+00:00
[2024-11-18T12:38:32.691+0000] {standard_task_runner.py:72} INFO - Started process 205 to run task
[2024-11-18T12:38:32.693+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline_dag', 'dbt_test', 'manual__2024-11-18T12:33:00.228556+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp5dho0lv4']
[2024-11-18T12:38:32.694+0000] {standard_task_runner.py:105} INFO - Job 19: Subtask dbt_test
[2024-11-18T12:38:32.732+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline_dag.dbt_test manual__2024-11-18T12:33:00.228556+00:00 [running]> on host 945969587f65
[2024-11-18T12:38:32.874+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:33:00.228556+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T12:33:00.228556+00:00'
[2024-11-18T12:38:32.876+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T12:38:32.923+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-18T12:38:32.924+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/airflow/etl_pipeline && dbt test --profiles-dir /opt/airflow/etl_pipeline']
[2024-11-18T12:38:32.941+0000] {subprocess.py:99} INFO - Output:
[2024-11-18T12:38:34.392+0000] {subprocess.py:106} INFO - [0m12:38:34  Running with dbt=1.8.8
[2024-11-18T12:38:34.621+0000] {subprocess.py:106} INFO - [0m12:38:34  Registered adapter: postgres=1.8.2
[2024-11-18T12:38:34.767+0000] {subprocess.py:106} INFO - [0m12:38:34  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-11-18T12:38:34.768+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2024-11-18T12:38:34.768+0000] {subprocess.py:106} INFO - - models.etl_pipeline.production
[2024-11-18T12:38:34.929+0000] {subprocess.py:106} INFO - [0m12:38:34  Found 6 models, 26 data tests, 2 sources, 539 macros
[2024-11-18T12:38:34.932+0000] {subprocess.py:106} INFO - [0m12:38:34
[2024-11-18T12:38:35.084+0000] {subprocess.py:106} INFO - [0m12:38:35  Concurrency: 1 threads (target='dev')
[2024-11-18T12:38:35.086+0000] {subprocess.py:106} INFO - [0m12:38:35
[2024-11-18T12:38:35.096+0000] {subprocess.py:106} INFO - [0m12:38:35  1 of 26 START test dbt_utils_expression_is_true_fact_revenue_minutes_to_service___0  [RUN]
[2024-11-18T12:38:35.248+0000] {subprocess.py:106} INFO - [0m12:38:35  1 of 26 PASS dbt_utils_expression_is_true_fact_revenue_minutes_to_service___0 .. [[32mPASS[0m in 0.15s]
[2024-11-18T12:38:35.252+0000] {subprocess.py:106} INFO - [0m12:38:35  2 of 26 START test dbt_utils_expression_is_true_fact_revenue_number_of_patient_visits___1  [RUN]
[2024-11-18T12:38:35.315+0000] {subprocess.py:106} INFO - [0m12:38:35  2 of 26 PASS dbt_utils_expression_is_true_fact_revenue_number_of_patient_visits___1  [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:35.319+0000] {subprocess.py:106} INFO - [0m12:38:35  3 of 26 START test dbt_utils_expression_is_true_fact_revenue_revenue___0 ....... [RUN]
[2024-11-18T12:38:35.393+0000] {subprocess.py:106} INFO - [0m12:38:35  3 of 26 PASS dbt_utils_expression_is_true_fact_revenue_revenue___0 ............. [[32mPASS[0m in 0.07s]
[2024-11-18T12:38:35.402+0000] {subprocess.py:106} INFO - [0m12:38:35  4 of 26 START test not_null_dim_doctors_department ............................. [RUN]
[2024-11-18T12:38:35.498+0000] {subprocess.py:106} INFO - [0m12:38:35  4 of 26 PASS not_null_dim_doctors_department ................................... [[32mPASS[0m in 0.09s]
[2024-11-18T12:38:35.506+0000] {subprocess.py:106} INFO - [0m12:38:35  5 of 26 START test not_null_dim_doctors_doctor ................................. [RUN]
[2024-11-18T12:38:35.598+0000] {subprocess.py:106} INFO - [0m12:38:35  5 of 26 PASS not_null_dim_doctors_doctor ....................................... [[32mPASS[0m in 0.09s]
[2024-11-18T12:38:35.604+0000] {subprocess.py:106} INFO - [0m12:38:35  6 of 26 START test not_null_dim_patients_patient_id ............................ [RUN]
[2024-11-18T12:38:35.670+0000] {subprocess.py:106} INFO - [0m12:38:35  6 of 26 PASS not_null_dim_patients_patient_id .................................. [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:35.676+0000] {subprocess.py:106} INFO - [0m12:38:35  7 of 26 START test not_null_dim_patients_patient_name .......................... [RUN]
[2024-11-18T12:38:35.738+0000] {subprocess.py:106} INFO - [0m12:38:35  7 of 26 PASS not_null_dim_patients_patient_name ................................ [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:35.742+0000] {subprocess.py:106} INFO - [0m12:38:35  8 of 26 START test not_null_fact_revenue_branch_id ............................. [RUN]
[2024-11-18T12:38:35.804+0000] {subprocess.py:106} INFO - [0m12:38:35  8 of 26 PASS not_null_fact_revenue_branch_id ................................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:35.809+0000] {subprocess.py:106} INFO - [0m12:38:35  9 of 26 START test not_null_fact_revenue_doctor_id ............................. [RUN]
[2024-11-18T12:38:35.874+0000] {subprocess.py:106} INFO - [0m12:38:35  9 of 26 PASS not_null_fact_revenue_doctor_id ................................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:35.879+0000] {subprocess.py:106} INFO - [0m12:38:35  10 of 26 START test not_null_fact_revenue_minutes_to_service ................... [RUN]
[2024-11-18T12:38:35.940+0000] {subprocess.py:106} INFO - [0m12:38:35  10 of 26 PASS not_null_fact_revenue_minutes_to_service ......................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:35.944+0000] {subprocess.py:106} INFO - [0m12:38:35  11 of 26 START test not_null_fact_revenue_number_of_patient_visits ............. [RUN]
[2024-11-18T12:38:36.004+0000] {subprocess.py:106} INFO - [0m12:38:36  11 of 26 PASS not_null_fact_revenue_number_of_patient_visits ................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.008+0000] {subprocess.py:106} INFO - [0m12:38:36  12 of 26 START test not_null_fact_revenue_patient_id ........................... [RUN]
[2024-11-18T12:38:36.069+0000] {subprocess.py:106} INFO - [0m12:38:36  12 of 26 PASS not_null_fact_revenue_patient_id ................................. [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.074+0000] {subprocess.py:106} INFO - [0m12:38:36  13 of 26 START test not_null_fact_revenue_revenue .............................. [RUN]
[2024-11-18T12:38:36.135+0000] {subprocess.py:106} INFO - [0m12:38:36  13 of 26 PASS not_null_fact_revenue_revenue .................................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.140+0000] {subprocess.py:106} INFO - [0m12:38:36  14 of 26 START test not_null_fact_revenue_risk_id .............................. [RUN]
[2024-11-18T12:38:36.204+0000] {subprocess.py:106} INFO - [0m12:38:36  14 of 26 PASS not_null_fact_revenue_risk_id .................................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.208+0000] {subprocess.py:106} INFO - [0m12:38:36  15 of 26 START test not_null_fact_revenue_row_id ............................... [RUN]
[2024-11-18T12:38:36.270+0000] {subprocess.py:106} INFO - [0m12:38:36  15 of 26 PASS not_null_fact_revenue_row_id ..................................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.277+0000] {subprocess.py:106} INFO - [0m12:38:36  16 of 26 START test relationships_fact_revenue_branch_id__branch_id__ref_dim_branches_  [RUN]
[2024-11-18T12:38:36.357+0000] {subprocess.py:106} INFO - [0m12:38:36  16 of 26 PASS relationships_fact_revenue_branch_id__branch_id__ref_dim_branches_  [[32mPASS[0m in 0.08s]
[2024-11-18T12:38:36.363+0000] {subprocess.py:106} INFO - [0m12:38:36  17 of 26 START test relationships_fact_revenue_doctor_id__doctor_id__ref_dim_doctors_  [RUN]
[2024-11-18T12:38:36.428+0000] {subprocess.py:106} INFO - [0m12:38:36  17 of 26 FAIL 6334 relationships_fact_revenue_doctor_id__doctor_id__ref_dim_doctors_  [[31mFAIL 6334[0m in 0.06s]
[2024-11-18T12:38:36.432+0000] {subprocess.py:106} INFO - [0m12:38:36  18 of 26 START test relationships_fact_revenue_patient_id__patient_id__ref_dim_patients_  [RUN]
[2024-11-18T12:38:36.495+0000] {subprocess.py:106} INFO - [0m12:38:36  18 of 26 PASS relationships_fact_revenue_patient_id__patient_id__ref_dim_patients_  [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.500+0000] {subprocess.py:106} INFO - [0m12:38:36  19 of 26 START test relationships_fact_revenue_risk_id__risk_id__ref_dim_risk_ . [RUN]
[2024-11-18T12:38:36.565+0000] {subprocess.py:106} INFO - [0m12:38:36  19 of 26 PASS relationships_fact_revenue_risk_id__risk_id__ref_dim_risk_ ....... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.569+0000] {subprocess.py:106} INFO - [0m12:38:36  20 of 26 START test source_not_null_staging_data_staging_patient_visits_doctor . [RUN]
[2024-11-18T12:38:36.701+0000] {subprocess.py:106} INFO - [0m12:38:36  20 of 26 PASS source_not_null_staging_data_staging_patient_visits_doctor ....... [[32mPASS[0m in 0.13s]
[2024-11-18T12:38:36.705+0000] {subprocess.py:106} INFO - [0m12:38:36  21 of 26 START test source_not_null_staging_data_staging_patient_visits_patient_id  [RUN]
[2024-11-18T12:38:36.761+0000] {subprocess.py:106} INFO - [0m12:38:36  21 of 26 PASS source_not_null_staging_data_staging_patient_visits_patient_id ... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.766+0000] {subprocess.py:106} INFO - [0m12:38:36  22 of 26 START test source_not_null_staging_data_staging_patient_visits_row_id . [RUN]
[2024-11-18T12:38:36.825+0000] {subprocess.py:106} INFO - [0m12:38:36  22 of 26 PASS source_not_null_staging_data_staging_patient_visits_row_id ....... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.829+0000] {subprocess.py:106} INFO - [0m12:38:36  23 of 26 START test source_not_null_trans_staging_patient_visits_patient_id .... [RUN]
[2024-11-18T12:38:36.889+0000] {subprocess.py:106} INFO - [0m12:38:36  23 of 26 PASS source_not_null_trans_staging_patient_visits_patient_id .......... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:36.893+0000] {subprocess.py:106} INFO - [0m12:38:36  24 of 26 START test source_unique_staging_data_staging_patient_visits_row_id ... [RUN]
[2024-11-18T12:38:36.972+0000] {subprocess.py:106} INFO - [0m12:38:36  24 of 26 PASS source_unique_staging_data_staging_patient_visits_row_id ......... [[32mPASS[0m in 0.08s]
[2024-11-18T12:38:36.976+0000] {subprocess.py:106} INFO - [0m12:38:36  25 of 26 START test unique_dim_patients_patient_id ............................. [RUN]
[2024-11-18T12:38:37.033+0000] {subprocess.py:106} INFO - [0m12:38:37  25 of 26 PASS unique_dim_patients_patient_id ................................... [[32mPASS[0m in 0.06s]
[2024-11-18T12:38:37.040+0000] {subprocess.py:106} INFO - [0m12:38:37  26 of 26 START test unique_fact_revenue_row_id ................................. [RUN]
[2024-11-18T12:38:37.108+0000] {subprocess.py:106} INFO - [0m12:38:37  26 of 26 PASS unique_fact_revenue_row_id ....................................... [[32mPASS[0m in 0.07s]
[2024-11-18T12:38:37.145+0000] {subprocess.py:106} INFO - [0m12:38:37
[2024-11-18T12:38:37.146+0000] {subprocess.py:106} INFO - [0m12:38:37  Finished running 26 data tests in 0 hours 0 minutes and 2.21 seconds (2.21s).
[2024-11-18T12:38:37.267+0000] {subprocess.py:106} INFO - [0m12:38:37
[2024-11-18T12:38:37.268+0000] {subprocess.py:106} INFO - [0m12:38:37  [31mCompleted with 1 error and 0 warnings:[0m
[2024-11-18T12:38:37.268+0000] {subprocess.py:106} INFO - [0m12:38:37
[2024-11-18T12:38:37.270+0000] {subprocess.py:106} INFO - [0m12:38:37  [31mFailure in test relationships_fact_revenue_doctor_id__doctor_id__ref_dim_doctors_ (models/transformation/schema.yml)[0m
[2024-11-18T12:38:37.271+0000] {subprocess.py:106} INFO - [0m12:38:37    Got 6334 results, configured to fail if != 0
[2024-11-18T12:38:37.272+0000] {subprocess.py:106} INFO - [0m12:38:37
[2024-11-18T12:38:37.273+0000] {subprocess.py:106} INFO - [0m12:38:37    compiled code at target/compiled/etl_pipeline/models/transformation/schema.yml/relationships_fact_revenue_8e6b53c87420f4a42087e24ffb6d0d96.sql
[2024-11-18T12:38:37.274+0000] {subprocess.py:106} INFO - [0m12:38:37
[2024-11-18T12:38:37.275+0000] {subprocess.py:106} INFO - [0m12:38:37  Done. PASS=25 WARN=0 ERROR=1 SKIP=0 TOTAL=26
[2024-11-18T12:38:38.837+0000] {subprocess.py:110} INFO - Command exited with return code 1
[2024-11-18T12:38:38.849+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-11-18T12:38:38.856+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=etl_pipeline_dag, task_id=dbt_test, run_id=manual__2024-11-18T12:33:00.228556+00:00, execution_date=20241118T123300, start_date=20241118T123832, end_date=20241118T123838
[2024-11-18T12:38:38.902+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T12:38:38.902+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 19 for task dbt_test (Bash command failed. The command returned a non-zero exit code 1.; 205)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-11-18T12:38:38.923+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-18T12:38:38.946+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-18T12:38:38.972+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
